subcommands/common.go

```diff
@@ -138,7 +138,7 @@ func DieNotNil(err error, message ...string) {
 			parts = append(parts, p)
 		}
 		parts = append(parts, err)
-		fmt.Println(parts...)
+		logrus.Error(parts...)
```

Comment:
Now on this...

I have already posted in the other PR that we use logs in debug debug mode, but not for regular user interaction.
Although colorful dialogs look cool and fancy, the control characters used to print them break some terminals.

I suspect that logrus is smart enough to only insert colors for terminals which support them.
Because, e.g. I've never seen these colors in my terminal.

Still, it is weird to see smth like `[ERRO] ERROR: your message goes here` in a regular (non-debug) command run.
I've tried that once as an excercise, and IIRC a team consunsus was that it looks strange.

So, `fmt.Println` for the savior. A `logrus` is nice for developers debugging cloud applications, but it does not fit all sizes.

Reply:
Hi @vkhoroz,

Indeed, I understand that you're right in noting:

```plaintext
I suspect that logrus is smart enough to only insert colors for terminals which support them.
```

Based on my research, Logrus has intelligence built-in to ensure it doesn't break terminals that can't handle ANSI sequences.

However, I noticed you mentioned:

```plaintext
Still, it is weird to see something like [ERRO] ERROR: your message goes here.
```

That is not caused by the usage of the lib itself. It is because we are adding `ERROR` and `WARNING` manually. This can be solved by:
```go
// Instead of this
logrus.Error("ERROR: ", err)
// Use this
logrus.Error(err)
```

Reply:
We do use https://github.com/fatih/color in some places to add some color; which also identifies terminal settings to some extent (this auto-detection is never ideal).
So, if it is about highliting `ERROR:` in some color, we could do it, if the team agrees.

The point was not to use `logrus` for something it was not created for.

Reply:
That is great I reopen this one when I saw the bump for fatih 
Now, I am checking this message so I think I have the answer for: https://github.com/foundriesio/fioctl/pull/284#issuecomment-1742602597

Cool. It is not a high priority and I am doing along side of the other things 
So, I will address this one in this direction asap I have time to work on it.

Thank you.

Reply:
I will close this one let's see